ci/templates/hosted-jobs.yml

```diff
@@ -13,18 +13,25 @@ jobs:
           poolName: "Azure Pipelines"
           os: macOS
           jdk: "1.11"
-        windows:
+        windows-cairo:
```

Comment:
these names should be descriptive. Misled by the name,  I had to check what it actually does in configuration file. I think better name could be `all-other`

Reply:
done in ed43dd5